README.md

```diff
@@ -461,10 +461,10 @@ OPTIONS
                                            be deployed
 
   --access-token=access-token              Eclipse Che OIDC Access Token. See the documentation how to obtain token:
-                                           https://www.eclipse.org/che/docs/che-7/authenticating-users/#obtaining-the-to
```

Comment:
Pls. rebase on master to get the latest README.md file

Reply:
This pr is build on top of the latest commit. If I checkout to master and do 'yarn run pack-binaries', README.md file is modified. I guess master branch has outdated README.md...

Reply:
And I didn't forget to make `git pull origin master`

Reply:
Ok, I see https://github.com/che-incubator/chectl/pull/847